data/colors/vars/component_light.ts

```diff
@@ -187,5 +187,24 @@ export default {
       bg: get('scale.blue.5'),
       disabledBg: get('scale.gray.5'),
     }
-  }
+  },
+
+  segmentedControl: {
+    bg: get('scale.gray.1'),
+
+    button: {
+      hover: {
+        bg: alpha(get('scale.gray.3'), 0.2),
+      },
+
+      active: {
+        bg: alpha(get('scale.gray.3'), 0.4),
+        border: get('scale.gray.3'),
+      },
+
+      selected: {
+        border: get('scale.gray.4'),
```

Comment:
I think we have to bump this to `scale.gray.5` because it doesn't hit the required 3:1 contrast ratio